### DISCUSSION TITLE
Updating to php-81-fpm-dev shows that xdebug.so can't be found

### DISCUSSION THREAD

**Discussion by etelford** [brefphp/bref#1484] [Support]
I have been running `php-80-fpm-dev` for quite some time and have had Xdebug working without issues. I am now attempting to update my `Dockerfile` and `docker-compose-yml` to point to `bref/php-81-fpm-dev` and `bref/php-81-fpm-dev:1.7.24`, respectively.

When I run `docker-compose up --build` the container builds, but I see this error:

```sh
NOTICE: PHP message: PHP Warning:  Failed loading Zend extension 'xdebug.so' (tried: /opt/bref/lib/php/extensions/no-debug-non-zts-20210902/xdebug.so (/opt/bref/lib/php/extensions/no-debug-non-zts-20210902/xdebug.so: cannot open shared object file: No such file or directory), /opt/bref/lib/php/extensions/no-debug-non-zts-20210902/xdebug.so.so (/opt/bref/lib/php/extensions/no-debug-non-zts-20210902/xdebug.so.so: cannot open shared object file: No such file or directory)) in Unknown on line 0
```

It's my understanding that all of the dev images come with Xdebug installed and as I said above, Xdebug works when I use `php-80-fpm-dev`. Any ideas of what could be happening here?

**Comment by mnapoli** [comment]
Indeed, xdebug should work in PHP 8.1 as well.

Could you try pulling (`docker pull bref/php-81-fpm-dev`) the latest images? I'm very surprised by the error, as it seems to look into a different directory for INI files.
